state/logs.go

```diff
@@ -66,6 +66,9 @@ var logIndexes = [][]string{
 	// the same time have a consistent ordering.
 	{"e", "t", "_id"},
 	{"e", "n"},
+
+	// This index is used for pruning logs by model and time.
+	{"e", "t"},
```

Comment:
Can we use a query hint instead to force the use of the e/t/id index?
Xtian made a comment on the bug that implied we could?